src/LiveComponent/doc/index.rst

```diff
@@ -2877,11 +2869,9 @@ To fix this, you have two options:
 
 \1) Make the ``key`` dynamic so it will be different after adding a new item:
 
-.. code-block:: twig
+.. code-block:: html+twig
 
-    {{ component('InvoiceLineItemForm', {
-        key: 'new_line_item_'~lineItems|length,
-    }) }}
+    <twig:InvoiceLineItemForm :key="'new_line_item_'~lineItems|length" />
```

Comment:
Does this work?

```suggestion
    <twig:InvoiceLineItemForm key="new_line_item_{{ lineItems|length }}" />
```

I wrote the pre-lexer and I honestly can't remember if that'll work or not. It sure looks nicer.